configs/scripts/program/build.sh

```
@@ -1,125 +1,38 @@
 #!/bin/bash
 
-set -euo pipefail
-
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 OUTPUT="./programs/.bin"
+# saves external programs binaries to the output directory
+source ${SCRIPT_DIR}/dump.sh ${OUTPUT}
+
 # go to parent folder
-cd "$(dirname "$(dirname "$(dirname "${SCRIPT_DIR}")")")"
+cd $(dirname $(dirname $(dirname ${SCRIPT_DIR})))
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)